src/main/kotlin/org/rust/lang/core/resolve2/FacadeResolve.kt

```diff
@@ -254,6 +261,13 @@ fun RsMetaItem.resolveToProcMacroWithoutPsi(): ProcMacroDefInfo? {
         is RsModInfo -> {
             val (_, defMap, modData) = info
             val macroPath = path?.pathSegmentsAdjustedForAttrMacro?.toTypedArray() ?: return null
+            if (macroPath.size == 1) {
```

Comment:
Lets instead pass `ResolveMode.IMPORT` to `defMap.resolvePathFp`, so that it will not ignore proc macros in legacy scope

Reply:
They're not equivalent. Here we resolve the last `ProcMacroDefInfo`, but `resolvePathFp` resolves the first (any) macro in the scope